src/main/resources/uk/gov/hmrc/scalatestaccessibilitylinter/accessibility-linter.conf

```diff
@@ -242,7 +242,7 @@ known-issues = [ # same format as hmrc/page-accessibility-check
   {
     tool = "axe"
     descriptionRegex = """Fix any of the following:[\n\r\s]+Some page content is not contained by landmarks"""
-    snippetRegex = """<a href=".*" class="govuk-back-link">Back</a>"""
+    snippetRegex = """<a href=".*" class="govuk-back-link.*".*>Back</a>"""
```

Comment:
it could almost be `""".*govuk-back-link.*"""` because the match is the description + the snippet so maybe the description + just the presence of the class govuk-back-link is enough to match this issue since I think back-links are only used in a fixed place